services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/sidecar.py

```diff
@@ -125,6 +126,13 @@ class DynamicSidecarSettings(BaseCustomSettings, MixinLoggingSettings):
 
     DYNAMIC_SIDECAR_R_CLONE_SETTINGS: RCloneSettings = Field(auto_default_from_env=True)
 
+    DYNAMIC_SIDECAR_AWS_S3_CLI_SETTINGS: AwsS3CliSettings = Field(
+        auto_default_from_env=True
+    )
+    DYNAMIC_SIDECAR_IS_RCLONE_ENABLED: bool = Field(
```

Comment:
I find it unnecessary complex.
DYNAMIC_SIDECAR_R_CLONE_SETTINGS is already optional. it will be None when RClone is not set.
You could go very simple using a validator that checks that both are not enabled at the same time or even prefer one or the other instead.